website-deep-dive.mdx

```diff
@@ -0,0 +1,145 @@
+
+- **Innovation** - We aimed for the website to reflect our identity as an innovative platform.
+- **Femininity** - Recognizing that many tech websites lean towards a masculine aesthetic, we aimed to bring a feminine touch to our brand, promoting inclusivity and diversity in the tech industry.
+- **Fun** - We wanted to infuse our website with a sense of playfulness and lightheartedness, ensuring that visitors feel engaged and entertained while navigating through our platform. This approach helps us break away from the typical seriousness often associated with tech websites, making our brand more approachable and relatable.
+
+As you'll see in the video below, our design choices reflect our brand's growth. The dark background represents innovation, showing we're forward-thinking. Colors like pink and purple adds a "feminity" touch, welcoming diversity in tech. Plus, our playful mascots in the video make sure visitors have fun and enjoy their time on our platform. Together, these elements show what our brand is all about.
+
+<ArticleVideo
+  platform="html"
+  src="https://us-east-1.storage.xata.sh/rp7qvd6get3vvd2hmjfqhuf0a8"
+  width="1604"
+  height="1080"
+/>
+
+### Color modes
+
+To refresh our brand we did a few adjustments to our color tokens. This had a ripple effect, influencing both our documentation and app. Additionally, we made a bold choice to exclusively design in dark mode, a decision we elaborated on in the "Navigating challenges" section.
+
+However, amidst these changes, we collectively decided to maintain two color modes for our documentation and app, prioritizing an exceptional Developer Experience (DX). This thoughtful decision ensures that our users have the freedom to select the color mode that aligns best with their preferences, thus elevating their overall journey with our platform.
+
+![Documentation color mode](/images/website-deep-dive/documentation-color-mode@2x.png)
+
+### Illustrations
+
+We also introduced new illustrations to our website, which we created using Figma. For each feature page, we designed unique hero illustration to represent the feature.
+We opted for a minimalist approach to ensure that the text remains the primary focus.
+
+![Feature pages hero images](/images/website-deep-dive/fp-hero-images@2x.png)
+
+We also created a set of illustrations to represent the different features of our platform.
+
+![Feature pages images](/images/website-deep-dive/fp-illustrations@2x.png)
+
+### Embracing fun and welcoming vibes
+
+As we explained earlier, one of the things we were eager to bring with our brand was "fun." We wanted to break away from seriousness a bit. And since our logo is a butterfly, we wanted it to somehow be present on our website and brand. We wanted to ensure our users felt warmly welcomed and included. That's why we decided to add a bit of fun into the mix without going overboard.
```

Comment:
```suggestion
As I explained earlier, one of the things we were eager to bring with our brand was "fun." We wanted to break away from seriousness a bit. And since our logo is a butterfly, we wanted it to somehow be present on our website and throughout our brand. We wanted to ensure our users felt warmly welcomed and included. That's why we decided to add a bit of fun into the mix without going overboard.
```